docs/contributing/code.rst

```diff
@@ -105,11 +105,9 @@ coverage will only be accepted with a very good reason; bug-fixing patches
 **must** demonstrate the bug with a test to avoid regressions and to check
 that the fix works.
 
-We have an IRC channel, our `django-cms-developers`_ email list,
-and of course the code reviews mechanism on GitHub - do use them.
+We have `a Slack group <https://www.django-cms.org/slack>`_, our `django-cms-developers`_
```

Comment:
Can we add our discourse forum http://discourse.django-cms.org/ here as the second communication channel and remove the email list (it's already removed from our website)? Afaik it was decided to only focus on Slack, Discourse and Stack overflow.